ext/php5/priority_sampling/priority_sampling.c

```diff
@@ -45,13 +46,19 @@ static void dd_update_upstream_services(ddtrace_span_fci *span, ddtrace_span_fci
 
     zval *service = ddtrace_spandata_property_service(&deciding_span->span);
 
+    zval *service_string;
+    ALLOC_INIT_ZVAL(service_string);
```

Comment:
no need to alloc a zval, just use a bare zval and zval_dtor it afterwards

Reply:
ack